homeassistant/components/decora_wifi/__init__.py

```diff
@@ -1 +1,61 @@
-"""The decora_wifi component."""
+"""Support for the myLeviton decora_wifi component."""
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+
+from .common import CommFailed, DecoraWifiPlatform, LoginFailed
+from .const import DOMAIN, PLATFORMS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass: HomeAssistant, config):
+    """Set up the Decora Wifi component."""
+    hass.data.setdefault(DOMAIN, {})
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up Decora Wifi from a config entry."""
+
+    conf_data = dict(entry.data)
```

Comment:
```suggestion
    conf_data = entry.data
```

Looks like you don't need to make a copy here